contracts/BMTFile.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./BMTChunk.sol";
+import "./TypedMemView.sol";
+
+contract BMTFile is BMTChunk {
+  using TypedMemView for bytes;
+  using TypedMemView for bytes29;
+
+
+// max segment count
+  uint256 public constant MAX_SEGMENT_COUNT = 128;
+
+  // chunk bmt levels
+  uint256 public constant CHUNK_BMT_LEVELS = 7;
+
+  struct ChunkInclusionProof{
+    uint64 span;
+    bytes32[] sisterSegments;
+  }
+
+  // Uses TypedMemView (DataView in JavaScript) to handle endianness
+  function getChunkSpanLength(
```

Comment:
can't we do something from the JS side to keep the parameters and the calculations in the smart contract simple?
Like convert the 8 bytes to Bigint or to number somehow to be interpreted by the EVM by default